crates/biome_css_analyze/src/keywords.rs

```diff
@@ -5422,6 +5424,25 @@ pub const LONGHAND_SUB_PROPERTIES_OF_SHORTHAND_PROPERTIES: [&[&str]; 57] = [
     ],
 ];
 
+// longhand => shorthand map
+pub(crate) static LONGHAND_SUB_PROPERTIES_MAP: LazyLock<HashMap<String, String>> =
+    LazyLock::new(|| {
+        let cap: usize = LONGHAND_SUB_PROPERTIES_OF_SHORTHAND_PROPERTIES
+            .map(|list| list.len())
+            .iter()
+            .sum();
+        let mut map: HashMap<String, String> = HashMap::with_capacity(cap);
+        for (shorthand, longhand_list) in SHORTHAND_PROPERTIES
+            .iter()
+            .zip(LONGHAND_SUB_PROPERTIES_OF_SHORTHAND_PROPERTIES)
+        {
+            for longhand in longhand_list {
+                map.insert((*longhand).to_string(), (*shorthand).to_string());
+            }
+        }
+        map
+    });
```

Comment:
This isn't very efficient, and we should not have it.

We are basically allocating a `String` for a string that we already have in memory. 

Considering how you use this constant, I think it's best to have a function called `is_longhand_sub_property` in the rule, which checks `SHORTHAND_PROPERTIES` and `LONGHAND_SUB_PROPERTIES_OF_SHORTHAND_PROPERTIES`

This comment is valid for `RESET_TO_INITIAL_PROPERTIES_MAP` too